### ISSUE TITLE
Do GBIF checklists not show taxa that do not match with the GBIF backbone?

### ISSUE THREAD

**Issue by sannegovaert** [trias-project/checklist-recipe#28]
I assumed that in GBIF, species names that aren’t linked to the Backbone would still appear in the published checklist - similar to occurrence datasets, where unmatched names/records are flagged with an orange circle. However, I haven’t been able to find an example of this yet.

See comment below:

===
I think the paragraph below needs a specific example, from e.g. The manual of Alien Plants. It's not very easy to understand if you're not familiar with GBIF. 

> The linking is based on the raw scientific name including authorship, the taxon rank and the provided taxonomy (eg kingdom). While GBIF keeps your raw data accessible on the published page, names that don't find a match in the GBIF Backbone won’t have active links to taxon web pages. To ensure your checklist is fully integrated and easy to navigate, your names should match as closely as possible with the GBIF Backbone.

_Originally posted by @LienReyserhove in https://github.com/trias-project/checklist-recipe/issues/23#issuecomment-4045615561_

**Comment by sannegovaert** [comment]
@damianooldoni could show us an [example](https://www.gbif.org/species/141264614)!

**Comment by damianooldoni** [comment]
Sure. See here below short script to get the unmatched taxa from the [Manual of the Alien Plants of Belgium](https://www.gbif.org/dataset/9ff7d317-609b-4c08-bd86-3bc404b77c42). As you can see there are 69 unmatched taxa. At the very end of the script, I show the keys of the first 10. Just add them to prefix https://www.gbif.org/species/ and you get to the specific page of the taxon. Example with very first key, `141264614`: https://www.gbif.org/species/141264614.

<img width="968" height="252" alt="Image" src="https://github.com/user-attachments/assets/94b51dc2-2d6b-442c-bc39-0eeadb94986d" />

``` r
library(rgbif)
library(purrr)
library(dplyr)

# Get taxa of Manual of the Alien Plants of Belgium
taxa <- rgbif::name_usage(
  datasetKey = "9ff7d317-609b-4c08-bd86-3bc404b77c42",
  limit = 20000
  ) |>
  purrr::pluck("data")
# Remove denormed classification, only original taxa
taxa <- taxa |>
  dplyr::filter(origin == "SOURCE")
# How many taxa with(out) match to GBIF Backbone?
taxa |> 
  dplyr::count(!is.na(nubKey)) |>
  dplyr::rename(match_backbone = `!is.na(nubKey)`)
#> # A tibble: 2 × 2
#>   match_backbone     n
#>   <lgl>          <int>
#> 1 FALSE             69
#> 2 TRUE            2548

# Taxa with no match to GBIF Backbone
unmatched <- taxa |>
  dplyr::filter(is.na(nubKey))

# Get keys of first 10 unmatched taxa
unmatched |> 
  head(10) |>
  dplyr::pull(key)
#>  [1] 141264614 141264666 141264835 141264849 141264916 141264923 141264957
#>  [8] 141264972 141265044 141265251
```

<sup>Created on 2026-03-30 with [reprex v2.1.1](https://reprex.tidyverse.org)</sup>
